lib/wallet/wallet.js

```diff
@@ -3956,6 +3954,16 @@ class Wallet extends EventEmitter {
     if (tx.getWeight() > policy.MAX_TX_WEIGHT)
       throw new Error('TX exceeds policy weight.');
 
+    const minFee = policy.getMinFee(
+      mtx.getVirtualSize(),
+      this.network.minRelay
+    );
+    const fee = mtx.getFee();
+    if (fee < minFee)
+      throw new Error('Fee is below minimum relay limit.');
+    if (fee > minFee * 10000)
```

Comment:
I think we need to make absurd fees little bit more configurable and move it to `policy`. It should default to this still, but allow easier future modifications.

Reply:
nit: maybe add more spacing around ifs.

Reply:
thanks, added in fa3729d7ae00ca3381affb0118336a21ca8173a8